xml-conduit/Text/XML/Stream/Parse.hs

```diff
@@ -604,29 +605,30 @@ contentMaybe = do
 content :: MonadThrow m => Consumer Event m Text
 content = fromMaybe T.empty <$> contentMaybe
 
--- | The most generic way to parse a tag. It takes a predicate for checking if
--- this is the correct tag name, an 'AttrParser' for handling attributes, and
--- then a parser for dealing with content.
+-- | The most generic way to parse a tag. It takes 
+-- a predicate for checking if this is the correct tag name and attributes, 
+-- an 'AttrParser' for handling attributes, 
+-- and then a parser for dealing with content.
 --
 -- This function automatically absorbs its balancing closing tag, and will
 -- throw an exception if not all of the attributes or child elements are
```

Comment:
This comment should be updated to describe the new behavior with respect to attributes.